docs/core/diagnostics/dotnet-counters.md

```diff
@@ -15,6 +15,12 @@ To install the latest release version of the `dotnet-counters` [NuGet package](h
 dotnet tool install --global dotnet-counters
 ```
 
+Starting from .NET 5, it is now possible to directly download a one-file executable for your operating system:
+
+- Win ([x86](https://aka.ms/dotnet-counters/win-x86) | [x64](https://aka.ms/dotnet-counters/win-x64) | [arm](https://aka.ms/dotnet-counters/win-arm) | [arm-x64](https://aka.ms/dotnet-counters/win-arm64))
+- OSX ([x64](https://aka.ms/dotnet-counters/osx-x64))
```

Comment:
`OSX` is an obsolete name.  Apple calls this `Mac OS`  we should do the same.

Reply:
macOS

Reply:
Good point: I'm updating the text to macOS (this "maxOS" syntax is the official Apple syntax)

Reply:
BTW, I've seen that "dotnet-dump` isn't supported on macOS." in the tool page but the OSX binary is available: is there any problem here?

Reply:
outdated docs :) cc @mikem8361

Reply:
> outdated docs :) cc @mikem8361

Do you want me to remove that line in the same PR?

Reply:
Sure -- one thing to mention would be that on macOS, dotnet-dump only works for target apps running .NET 5.0+.

Thanks @chrisnas!

Reply:
dotnet-dump is supported on MacOS for 5.0 runtimes. ELF core dumps are generated by `collect` and read by `analyze`.  MachO dumps are not supported.